2020/06-16.md

```diff
@@ -0,0 +1,16 @@
+
+# Agenda for Ecma TC39 Incubator Call 2020-06-16
+
+- **Dates**: June 16, 2020
+- **Time**: 9:00 to 10:00 PDT
+- **Attendee information**: [Reflector](https://github.com/tc39/Reflector/issues/297)
+
+## Agenda topic rules
+
+Please add topics you would like from the proposals from the [current charter](https://github.com/tc39/incubator-agendas/issues/5).
+
+- The champion deadline is 1:00 June 10, 2020.
+
+## Agenda items
+
+* An update on UUID ~ Benjamin Coe
```

Comment:
What we learned from the first incubator call was it'd be really helpful if champions could help scope the conversation by asking for feedback on specific items. Could you add one or two specific topics?

Reply:
@syg I would be happy to flesh out the agenda a bit, is the request that I essentially rough out an agenda? If so, should we plan on the whole hour, or will there be a few topics of conversation?

Reply:
You got the floor for the whole hour. :)

Reply:
I've merged it for now for visibility. I'll add you as a contributor to edit the agenda directly.